setup.py

```diff
@@ -57,15 +57,12 @@
 def openmp_build_ext():
     from numpy.distutils.command.build_ext import build_ext
 
-    compile_flags = ['-fopenmp']
-    link_flags = ['-fopenmp']
-
     code = """#include <omp.h>
     int main(int argc, char** argv) { return(0); }"""
```

Comment:
This could be refactored further and we could remove the function entirely

Reply:
could you please explain how?

Reply:
@emmanuelle See https://github.com/scikit-image/scikit-image/pull/4924/commits/085e8008e345676c9b78c17eacafb657b2b4aa08

Essentially we group the import with the remaining imports, make the code string local to the class, remove the wrapping function and pass the class to `build_ext`